src/Migrations/Version20210422065835.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
Is it normal to have a migration of Sylius core in this commit?

Reply:
It's true that it is weird but if I remove this migration then `symfony console doctrine:schema:validate` return an error

Reply:
I think you need to check the doctrine configuration with MariaDB. 
A pull request talk about this thing on Sylius : 
https://github.com/Sylius/Sylius/pull/12413